setup/install/environment.md

```diff
@@ -67,6 +67,54 @@ hal config deploy edit --type distributed --account-name $ACCOUNT
 This command changes the type of the next deployment of Spinnaker, and will
 deploy it to the account you have previously configured.
 
+## Local Git
+
+The __Local Git__ installation means Spinnaker will be cloned, built and run on
+the single machine Halyard is run on.
+
+### Intended Use-case
+
+The __Local Git__ installation is intended for developers who want to contribute
+to Spinnaker. It is not intended to be used to manage any production environment.
+
+### Prerequisites
+
+#### Install local dependancies
+
+Ensure that the following are installed on your system
```

Comment:
Colon after "...the following are installed on your system"

Reply:
Done.